content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -605,13 +622,9 @@ For example:
         values: ["cluster-services"]
 ```
 
-See [LimitedResources](https://github.com/kubernetes/kubernetes/pull/36765) and [Quota support for priority class design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-priority-resourcequota.md) for more information.
-
-## Example
-
-See a [detailed example for how to use resource quota](/docs/tasks/administer-cluster/quota-api-object/).
-
 ## {{% heading "whatsnext" %}}
 
 - See [ResourceQuota design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_resource_quota.md) for more information.
-
+- See a [detailed example for how to use resource quota](/docs/tasks/administer-cluster/quota-api-object/).
+- Read [Quota support for priority class design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-priority-resourcequota.md).
+- See [LimitedResources](https://github.com/kubernetes/kubernetes/pull/36765)
```

Comment:
Is there a good reason to include a reference to a pull request?

Reply:
Well ... good question. I think I relocated that link from the previous section.
Do you want me to remove it?

Reply:
Not sure whether the pull request is remaining as documentation or will be deleted in the future?
Can defer removing until later?

Reply:
My general feeling is that as a development community, people tend to treat issues, PRs, docs and even code equally. They all serve communication channels. The kubernetes community is more unconstrained on this. Teams use Google docs, slack channel, Google forum extensively. Sometimes it is pretty difficult to build a clear understanding of the whole picture.

Reply:
OK